app/models/booking.rb

```diff
@@ -1,4 +1,6 @@
 class Booking < ApplicationRecord
   belongs_to :user
   belongs_to :product
+  enum status: [:available, :pending, :not_available]
+  validates :rating, inclusion: { in: (1..10)}
```

Comment:
it might be better to change the rating system from 1~10 to 1~5 in terms of cosmetics. to be more specific, i think 5 star rating systems look more appealing than 10 star rating systems

Reply:
I agree